### PR TITLE
index-html-font-and-title-update

### PR DESCRIPTION
**Pull request by ravi0900**
hacktoberfest, merge please

### CODE REVIEW

**Comment by vaibhav-1508** [comment]
Can you please paste the screenshots of changes that you made?

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,18 @@
...
   <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (vaibhav-1508) [inline public/index.html]: vite.svg is missing in the root folder

**Review by vaibhav-1508 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (ravi0900) [reply public/index.html]: i have removed this 🙂!

**Comment by ravi0900** [comment]
@vaibhav-1508 please review it again and merge it

**Comment by vaibhav-1508** [comment]
The branch has conflicts

**Comment by ravi0900** [comment]
> The branch has conflicts

how to resolve this conflict can you help, i newbie and nub 😥

**Comment by ravi0900** [comment]
@vaibhav-1508 I think now fix !

File: public/index.html [hunk 2]
```diff
@@ -1,43 +1,18 @@
...
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <meta charset="UTF-8" />
+   
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Expense Tracker</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (vaibhav-1508) [inline public/index.html]: The fonts imported here haven't been applied in the CSS

Comment (vaibhav-1508) [reply public/index.html]: Make changes in the CSS files

**Review by vaibhav-1508 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (ravi0900) [reply public/index.html]: So made the changes, also added nice background color as button color, looks nice

**Comment by ravi0900** [comment]
image

**Comment by vaibhav-1508** [comment]
I was about to reopen the PR but you deleted the project
image

**Comment by ravi0900** [comment]
> I was about to reopen the PR but you deleted the project image

It's ok I'm not that good, thanks for removing invalid label and I'm very sorry for not able to provide you a meaningful contribution.

**Pull request closed without merging**